JSON/very_small_example_report.json

```diff
@@ -0,0 +1,75 @@
+{
+	"title": "OSCAP Scan Result",
+	"identity": "jrodak",
+	"profile_name": "xccdf_com.example.www_profile_test_single_rule",
+	"platform": "",
+	"target": "rh-hony",
+	"cpe_platforms": [],
+	"scanner": "OpenSCAP",
+	"scanner_version": "1.3.5",
+	"benchmark_url": "#scap_org.open-scap_comp_test_single_rule.xccdf.xml",
+	"benchmark_id": "xccdf_com.example.www_benchmark_dummy",
+	"benchmark_version": "1.0",
+	"start_time": "2021-11-30T12:27:51+01:00",
+	"end_time": "2021-11-30T12:27:51+01:00",
+	"test_system": "cpe:/a:redhat:openscap:1.3.5",
+	"score": 100.0,
+	"score_max": 100.0,
+	"rules": {
+		"xccdf_com.example.www_rule_test-pass": {
+			"rule_id": "xccdf_com.example.www_rule_test-pass",
+			"title": "This rule always passes",
+			"result": "pass",
+			"multi_check": false,
+			"time": "2021-11-30T12:27:51+01:00",
+			"severity": "Unknown",
+			"identifiers": [],
+			"references": [],
+			"description": "Description",
+			"rationale": "",
+			"warnings": [],
```

Comment:
There are different types of warning in the content, like `general`, `performance`, `functionality` and `regulatory`. And the types can be repeated in a rule. So, the json output should identify the `type` and the message itself. Could you extend this?

Reply:
Yes, I can extend this.